tests/run/paramForwarding_together.scala

```diff
@@ -0,0 +1,29 @@
+class A(val member: Int) {
+  def getAMember = member
+}
+
+class SubA(member: Int) extends A(member) {
+  def getSubAMember = member
+}
+
+class B(member: Int) extends SubA(member) {
+  def getBMember = member
+}
+
```

Comment:
I am not yet convinced this fixes it. Can you also add test cases where A, SubA, and B appear in different orders? Like: B, SubA, A, for instance?